code/__DEFINES/obj_flags.dm

```diff
@@ -29,6 +29,7 @@
 #define IN_STORAGE				(1<<11) //is this item in the storage item, such as backpack? used for tooltips
 #define SURGICAL_TOOL			(1<<12)	//Tool commonly used for surgery: won't attack targets in an active surgical operation on help intent (in case of mistakes)
 #define EYE_STAB				(1<<13) /// Item can be used to eyestab
+#define HAND_ITEM				(1<<14) /// If an item is just your hand (circled hand, slapper) and shouldn't block things like riding
```

Comment:
You are not wrong in that this is an useless flag, given the commit name.
That is so because it's not a generic property. Maybe try one of the other suggestions if you can't be bothered to come up with one?

Reply:
Fake item doesn't really make sense because what does fake mean? They're still items that have their own behavior, and represent your wholly real hand. Changeling flesh disguise items and fake armblades are far more "fake" than these are despite still making sense to block being able to hold onto borgs.

Visual only doesn't make sense either since they still have behavior tied to them, and in the case of the circle hand, it can deal actual damage if the correct conditions are filled. Neither of them even have in-hand sprites (and I think ABSTRACT stops them from even showing up in examine), so they actually have less visuals than the average item, so that name doesn't fit either

ABSTRACT would be the closest descriptor for them, but that flag is used by wholly tangible things like chainsaw arms and armblades that wound indeed prevent holding on while riding a borg in a way that making a circle with your hand wouldn't.

In the end, the only common ground I found between the circle hand and the slap hand is the fact that they're just your hand making a certain shape, which is why they shouldn't block things like holding a real object would, and that they're made by doing an emote. There is no general mechanic to tease out here that can be expanded upon, these two items are the only ones that make sense for this exception to apply to. If you can come up with one that fits just these two items while still being more generic, I'll so that instead

Reply:
alright